fullstack/README.md

```diff
@@ -0,0 +1,64 @@
+This is a coding exercise to help assess candidates looking to join our team at Condé Nast International as FullStack Developers.
+
+The test is intended to be completed at home in no more than 1-2 hours. It is not timed, so if you want to take more time to showcase your coding skills and ability you may do so (note this is not mandatory, nor expected).
+
+If you haven't finished in that time, you can send us your progress - we don't want you to spend days on this. We understand that your time is valuable, and appreciate any time that you contribute towards building a strong team here.
+
+If you **really cannot spare the time**, you may want to take a look at Option 2.
+
+## Option 1
+
+Build a JavaScript application that shows the latest news from the United Kingdom, using the [newsapi.org](https://newsapi.org) service.
+
+Running the app we should be able to see the latest **UK news** and give the ability to the user to filter them by categories. Given the fact that this is a FullStack challenge, we are expecting a Frontend application and a Backend application that communicate with each other.
+
+A FullStack developer is someone with familiarity in each layer; you might be stronger on the frontend, or vice-versa. Feel free to focus more on what you know best, be creative and show us your skills!
```

Comment:
Is this task enough for someone to show their style? How would you see a backend engineer showing their senior ability on this task? Same question for front end. Perhaps we could have suggestions for a front end or back end option? e.g. For front end - also render an article page and format some basic html from the body. For back end - build a solution to reduce number of stale API calls